### PR TITLE
Snowflake Beret for the FC + admeme headband

### PR DESCRIPTION
**Pull request by Alterist**
🆑 
add: The Field Commander now has his own special armored beret and swanky black gloves
/ 🆑  
Gives the Field Commander a special beret and gloves, also created a snake bandana subtype

### CODE REVIEW

**Comment by DominikPanic** [comment]
Could you make the in-game item for it? Doesn't have to be any different, just the same beret with redefined icon_state. Current policy unfortunately is to not merge just assets without the accompanying code. :/

**Comment by Alterist** [comment]
done

File: code/modules/clothing/head/head.dm [hunk 1]
```diff
@@ -43,20 +42,24 @@
...
 	name = "\improper Command Master at Arms beret"
 	desc = "A beret with the lieutenant insignia emblazoned on it. It shines with the glow of corrupt authority and a smudge of doughnut."
 	icon_state = "beretwo"
 	armor = list(melee = 60, bullet = 80, laser = 80,energy = 20, bomb = 10, bio = 0, rad = 0)
 
+/obj/item/clothing/head/tgmcberet/fc
+    name = "\improper Field Commander beret"
+    desc = "A beret with the field commander insignia emblazoned on it. It commands loyalty and bravery in all who gaze upon it."
+    icon_state = "beretfc"
+    armor = list(melee = 60, bullet = 80, laser = 80,energy = 20, bomb = 10, bio = 0, rad = 0)
```
Comment (Rohesie) [inline code/modules/clothing/head/head.dm]:
```suggestion
    armor = list("melee" = 60, "bullet" = 80, "laser" = 80, "energy" = 20, "bomb" = 10, "bio" = 0, "rad" = 0, "fire" = 20, "acid" = 20)
```
Spooky's PR changed the armor paths.

Comment (Alterist) [reply code/modules/clothing/head/head.dm]: done

**Comment by Rohesie** [comment]
Travis check failing.

**Comment by spookydonut** [comment]
```
including code/modules/shuttles/shuttles_multi.dm
including code/modules/supermatter/supermatter.dm
mv: cannot stat ‘tgmc.mdme.dmb’: No such file or directory
mv: cannot stat ‘tgmc.mdme.rsc’: No such file or directory
The command "tools/travis/build_byond.sh" exited with 1.
```
travis check interupted or something

File: code/modules/clothing/head/head.dm [hunk 2]
```diff
@@ -122,9 +124,13 @@
 /obj/item/clothing/head/headband/rambo
 	name = "headband"
 	desc = "It flutters in the face of the wind, defiant and unrestrained, like the man who wears it."
-	icon = 'icons/obj/clothing/hats.dmi'
 	icon_state = "headband_rambo"
-	sprite_sheet_id = 0
+
+/obj/item/clothing/head/headband/snake
+	name = "headband"
+	desc = "A replica of the headband of a legendary soldier. Sadly it doesn't offer infinite ammo"
```
Comment (Rohesie) [inline code/modules/clothing/head/head.dm]:
```suggestion
	desc = "A replica of the headband of a legendary soldier. Sadly it doesn't offer infinite ammo. Yet."
```

Comment (Rohesie) [reply code/modules/clothing/head/head.dm]: More to force another travis check than anything else.

**Comment by DominikPanic** [comment]
@spookydonut CTRL+F error for the future
`code/modules/clothing/head/head.dm:61:error: inconsistent indentation`

**Pull request merged**